src/classes/services/ContactService.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactService {
```

Comment:
MAJOR
Service class should have `inherited sharing` as it has to inherit it from caller.

Reply:
Don't know ab inherited sharing. Thanks!